app/src/main/shared/components/Form.tsx

```diff
@@ -76,22 +79,33 @@ export class Form extends React.Component<FormProps, FormState> {
                 disabled: true
             });
 
-            this.submitForm()
+            this.submitForm(form)
         }
     }
 
+    onChange(){
+        this.setState({
+            errors: [],
+            hasSuccess: false
+        });
+    }
```

Comment:
doing this on the form change event so that alerts get cleared when the form inputs change